setup.py

```diff
@@ -72,33 +60,31 @@ def run(self):
     'pytest',
 ]
 
-setup(
-    # Metadata
-    name='autogluon',
-    version=version,
-    author='AutoGluon Community',
-    url='https://github.com/awslabs/autogluon',
-    description='AutoML Toolkit with MXNet Gluon',
-    long_description=long_description,
-    long_description_content_type='text/markdown',
-    license='Apache',
+if __name__ == '__main__':
+    create_version_file()
```

Comment:
Just to make sure I fully understand, the addition of this line is what actually fixes the bug, correct? 
My understanding is the bug existed before because `create_version_file()` was never called outside `python setup.py develop` or `python setup.py install`